doc/source/whatsnew/v3.0.0.rst

```diff
@@ -96,6 +96,13 @@ Deprecations
 - Deprecated :meth:`Timestamp.utcnow`, use ``Timestamp.now("UTC")`` instead (:issue:`56680`)
 -
 
+.. ---------------------------------------------------------------------------
+.. _whatsnew_300.prior_deprecations:
+Removal of prior version deprecations/changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Removed the :class:`ArrayManager` (:issue:`55043`)
```

Comment:
```suggestion
- Removed the ``ArrayManager`` (:issue:`55043`)
```

(this will no longer reference anything)

Reply:
Done.